backend/video-transcoding-server/src/services/fs/fs-service.ts

```diff
@@ -0,0 +1,16 @@
+import fs from 'fs';
+
+export class FsService {
+  public static createFolder({ path }: { path: string }): void {
+    if (!fs.existsSync(path)) {
+      fs.mkdirSync(path, { recursive: true });
```

Comment:
FYI: All operations with the file system would be better execute asynchronously.

Reply:
in this case i think we should block loop until the folder is created